### ISSUE TITLE
Proposal: Albums, virtual folders or tags

### ISSUE THREAD

**Issue by Eleksar387** [RupertAvery/DiffusionToolkit#55]
It would be great to have some kind of possibility to categorize images. So if you work on project you can add images to that project.

**Comment by RupertAvery** [comment]
Yes, Albums is a planned feature. I just need to think of where to place the albums and how to access / display them.  I'm quite sure that I want to add a right-click context menu that will add to album and allow you to either create an album or select from a list of top 10 recent albums or something.

**Comment by RupertAvery** [comment]
Current album work in progress:

![image](https://user-images.githubusercontent.com/1910659/232360658-eb16971d-d1f5-4139-809b-7581bf51d4c9.png)

![image](https://user-images.githubusercontent.com/1910659/232360714-95175b8d-edd7-4582-a1da-6c93c8a91c18.png)

It's working fine, just needs some usability stuff, like, 

1) limiting the number of albums in the right-click menu
2) moving the latest album to the top of the right-click menu
3) displaying a full list of albums in a dialog or something when you have more than x number of albums

**Comment by Eleksar387** [comment]
Great news. Honestly I thought your abandoned the project.

**Comment by RupertAvery** [comment]
Just a little hiatus.  It tends to consume a lot of my time, and I needed to step back and take a little break
